EmployeeManagement/Tavisca.WCF.DAL/FileSystem.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.ServiceModel;
+using System.Text;
+using System.IO;
+namespace Tavisca.WCF.DAL
+{
+    public class FileSystem
+    {
+        public void SaveEmployee(string jsonString, string id)
```

Comment:
Where did you get the json string. Serialization and deserialization should be done in the DAL. If tomorrow I change the DAL to use DB, I don't have to do the serialization, So I'll have to change the contract.